src/editor/braneProject.cpp

```diff
@@ -110,12 +110,16 @@ void BraneProject::initLoaded()
 {
     refreshAssets();
 
-    Json::Value& assets = _file.data()["assets"];
+    // Create required project data directories.
+    const auto projectRootDirectory = projectDirectory();
+    std::filesystem::create_directories(projectDirectory() / "assets");
+    std::filesystem::create_directories(projectDirectory() / "cache");
+
     if(!_file.data().isMember("assetIdCounter"))
         _file.data()["assetIdCounter"] = 0;
 
     _fileWatcher = std::make_unique<FileWatcher>();
-    _fileWatcher->loadCache(projectDirectory() / "cache" / "changeCache");
+    _fileWatcher->loadCache(projectDirectory() / "cache" / "changeCache.bin");
```

Comment:
This will break existing caches but it's clearer that this is a binary cache file, would you like to keep this?